draft-ietf-quic-load-balancers.md

```diff
@@ -188,10 +194,9 @@ configuration until it takes precautions to make sure that all connections using
 CIDs with an old configuration at that codepoint have closed or transitioned.
 
 Servers MUST NOT generate new connection IDs using an old configuration after
-receiving a new one from the configuration agent. Servers MUST send
-NEW_CONNECTION_ID frames that provide CIDs using the new configuration, and
-retire CIDs using the old configuration using the "Retire Prior To" field of
-that frame.
+receiving a new one from the configuration agent. Servers MUST use that QUIC
+version's methods to update the client with CIDs (e.g., NEW_CONNECTION_ID
+frames) using the new configuration and retire CIDs using the old configuration.
```

Comment:
Don't you request that your peer retires them?  Well, in QUIC v1 and v2 you do.  Who knows otherwise :)